pkg/vm/vm.go

```diff
@@ -1860,7 +1860,7 @@ func (v *VM) ContractHasTryBlock() bool {
 		if ictx.sc != topctx.sc {
 			return false // Different contract -> no one cares.
 		}
-		for j := 0; j < ictx.tryStack.Len(); j++ {
+		for j := range ictx.tryStack.Len() {
```

Comment:
Seems to be OK here, but notice that we do have some fancy loops in exception handling that can't be converted.

Reply:
what do you mean?